### DISCUSSION TITLE
Azure app service?

### DISCUSSION THREAD

**Discussion by philliaj** [atifaziz/NCrontab#119] [General]
We are using this library to run some services in a azure app service hosted environment. For some reason the scheduled jobs are entirely unreliable. Will fire sometimes but mostly they do not fire on the assigned cron schedule. The app services is set to always on...

wondering if there are any suggestions for this - thanks!

**Comment by atifaziz** [comment]
It seems to me that this is more a question for Azure App Service than NCrontab since the latter doesn't inherently do any scheduling or lifetime management. It's just a library for calculation occurrence of a schedule expressed in crontab format.
